### DISCUSSION TITLE
Swipe down to access the tagging feature

### DISCUSSION THREAD

**Discussion by kmcbest** [deckerst/aves#1048] [Ideas]
I don't you about others but the feature I use most is the tagging feature, but accessing it requires to place the tagging icon on the quick actions (both in thumbnails and the viewer) then click it.

I noticed if you swipe down on an image in viewer mode you're back to the thumbnail page. But since Android already has a "back" action (screen edge swipe) or physical back button (if you're using old models), this "swipe down" action is redundant and I hardly use it. How about changing it to open the tagging page?

**Comment by lordofscripts** [comment]
Same here. To me Tagging and Geolocating are the most used features to organize them.

**Comment by deckerst** [comment]
I don't intend to change the current behaviour. The swipe down gesture to leave the image is a common and expected gesture.
To you tagging is important, to others it may be useless. And allowing customization of every single gesture is a no-go.

How is the tagging quick action unsuitable? It's a tap instead of a swipe.
Also, a long press on it lets you tag with your most recent tags without leaving the page.

**Reply by kmcbest** [reply]
> I don't intend to change the current behaviour. The swipe down gesture to leave the image is a common and expected gesture. To you tagging is important, to others it may be useless. And allowing customization of every single gesture is a no-go.

This I understand.

> How is the tagging quick action unsuitable? It's a tap instead of a swipe. Also, a long press on it lets you tag with your most recent tags without leaving the page.


About that, the most recent tag is "one at a time" and the list is 5 tags long, which is relatively short. When tagging anime images like we do on booru-style site, we often apply 3~5 tags to an image before clicking save. So can it be changed to "recent tagging action"? I mean if i apply three tags to an image on my last action, I can re-apply those three tags in one go from the "recent" menu on the next image.
